### PR TITLE
Translations for the additives taxonomy

### PR DESCRIPTION
**Pull request by teolemon**
Translations for the additives taxonomy
A first review pass would be welcome

### CODE REVIEW

File: taxonomies/additives.txt [hunk 1]
```diff
@@ -2140,8 +2140,12 @@ nl:E160b, — annatto, bixine, norbixine, Orleaan, Anatto, Annato
 nl_be:E160b, — annatto, bixine, norbixine, Orleaan, Anatto, Annato
 pl:E160b, Annato, biksyna, norbiksyna, Annatto
 pt:E160b, Anato, bixina, norbixina, Colorau, Coloral, Urucu
+<<<<<<< Updated upstream
```
Comment (stephanegigandet) [inline taxonomies/additives.txt:2143]:
```suggestion
<<<<<<< Updated upstream
```

Comment (stephanegigandet) [inline taxonomies/additives.txt:2143]: unresolved conflict

File: taxonomies/additives.txt [hunk 2]
```diff
@@ -2140,8 +2140,12 @@ nl:E160b, — annatto, bixine, norbixine, Orleaan, Anatto, Annato
...
 ru:E160b, Annatto, аннато, экстракт анато
+=======
+>>>>>>> Stashed changes
```
Comment (teolemon) [inline taxonomies/additives.txt:2146]:
```suggestion
```

File: taxonomies/additives.txt [hunk 3]
```diff
@@ -2140,8 +2140,12 @@ nl:E160b, — annatto, bixine, norbixine, Orleaan, Anatto, Annato
...
 ro:E160b, Annatto, bixină, norbixină (i)
+ru:E160b, Annatto, аннато
```
Comment (teolemon) [inline taxonomies/additives.txt:2148]:
```suggestion
ru:E160b, Annatto, аннато, экстракт анато
```

**Review by stephanegigandet — APPROVED** [review]

**Pull request merged**